shipyard_blueprints/hubspot/shipyard_hubspot/hubspot_utils.py

```diff
@@ -235,6 +235,15 @@ def handle_import_file(
                             column_type="HUBSPOT_ALTERNATE_ID",
                         )
                     )
+                elif header == "hs_object_id":
```

Comment:
Should this also be a global variable? Not familiar with HubSpot but it would be good for readability in this code.